test/functional/services/monaco_editor.ts

```diff
@@ -62,8 +62,6 @@ export class MonacoEditorService extends FtrService {
         nthIndex,
         value
       );
-    });
-    await this.retry.try(async () => {
```

Comment:
This was failing while running my functional test because it tries to modify the Monaco editor after a refresh before it's initialized. I updated it to retry the whole update instead of just the check, and ran the FTR to be sure.